Gemfile

```diff
@@ -46,6 +46,8 @@ gem 'mediawiki-page-replaceable_content'
 # Report exceptions to maintainers
 gem 'exception_notification'
 
+gem 'membership-comparison', path: '../membership-comparison'
```

Comment:
I think the tests are failing because this is pointing to a path, rather than a `github:` repo 🙂

Reply:
Oops, yes of course, I've pushed a fixup